src/main/webapp/app/controllers/appLoginController.js

```diff
@@ -1,6 +1,6 @@
 sage.controller('AppLoginController', function ($controller, $scope) {
 
-  angular.extend(this, $controller('AuthenticationController', {
+  angular.extend(this, $controller('LoginController', {
```

Comment:
This is correct as AppLoginController is intended to extend weaver LoginController. However, this leaves external login broken. The following changes to the loginModal.html are needed:

remove ng-controller="AppLoginController" from the top level and add it to the form here:
https://github.com/TAMULib/SAGE/blob/master/src/main/webapp/app/views/modals/loginModal.html#L9

add ng-controller="AuthenticationController" here:
https://github.com/TAMULib/SAGE/blob/master/src/main/webapp/app/views/modals/loginModal.html#L37